docs/contributing/tests.md

```diff
@@ -50,19 +50,29 @@ samples are provided. To run the tests, follow this procedure:
 
   ```
   sudo dnf -y install python-devel
-  python3 -m venv venv
-  source venv/bin/activate
-  pip install openstackclient osc_placement jmespath
+  pip install --user openstackclient osc_placement jmespath
```

Comment:
Hmm at least personally i'd probably not want to use this over the virtualenv... Maybe we should provide more options here? E.g. if someone is using Fedora + venv vs. CentOS + non-venv? At least for Adoption we've been using Fedora (38) for the lab machines.

Reply:
Is there a reason to avoid virtualenv?